lib/daru/date_time/index.rb

```diff
@@ -366,6 +366,24 @@ def [] *key
       end
     end
 
+    def pos *args
+      # to filled
+      out = self[*args]
+      return out if out.is_a? Numeric
+      out.map { |date| self[date] }
+    end
+
+    def subset *args
+      self[*args]
+    end
+
+    def respond? *args
```

Comment:
what does this method do?
 
In Ruby, `respond_to?` is a method that tells you if a particular method is defined on an object. This does not look like something similar.